src/sass/post-911-gib-status.scss

```diff
@@ -32,3 +32,28 @@ hr {
   	margin-top: 2em;
   }
 }
+
+.print-status {
```

Comment:
I think this is a reasonable approach given that we can't have different static page layouts within the same React app. Another approach I've taken in the past is to create a class that can be added/removed on the `html` element that sets the header and footer inside it to `display: none`. I don't think that's better than what you have, but it's another option if this has unexpected issues.